blogs/2016/08/19/goodbyeuservoice.md

```diff
@@ -0,0 +1,94 @@
+---
+Order: 14
+TOCTitle: Goodbye User Voice
+PageTitle: Goodbye User Voice
+MetaDescription: VS Code is closing User Voice in favor of Github reactions.  
```

Comment:
Use "Visual Studio Code" in the MetaDescription.
Should be "GitHub" everwhere in the doc.